Sources/Widgets/SVG/SVGLandmarkRepresentation/index.js

```diff
@@ -11,6 +11,22 @@ function vtkSVGLandmarkRepresentation(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkSVGLandmarkRepresentation');
 
+  publicAPI.setDy = (val) => {
+    model.dy = val;
+  };
+
+  publicAPI.getDy = () => {
+    return model.dy;
+  };
```

Comment:
use macro.setGet to create those methods

Reply:
the problem with set/get macro is to my mind that it modifies data stored only in default values. My problem here was to allow multiple instances of SVGLandmarkRepresentation to have different values of Dy for each instance (and avoid a bug of moving all instances at once when only one instance was to be moved)

Reply:
Not true. Data are stored inside `model` that is specific to each instance.